gap/RREF.g

```diff
@@ -0,0 +1,35 @@
+IsRREF := function(A)
```

Comment:
I suggest `IsMatrixInRREF` as a more descriptive name. GAP has loads of objects.